AttributeError: 'dict' object has no attribute 'loss'